tests/delete/tests.py

```diff
@@ -6,14 +6,20 @@
 
 from .models import (
     MR, A, Avatar, Base, Child, HiddenUser, HiddenUserProfile, M, M2MFrom,
-    M2MTo, MRNull, Parent, R, RChild, S, T, User, create_a, get_default_r,
+    M2MTo, MRNull, Parent, R, RChild, S, T, Ticket30333, User, create_a,
+    get_default_r,
 )
 
 
 class OnDeleteTests(TestCase):
     def setUp(self):
         self.DEFAULT = get_default_r()
 
+    def test_30330(self):
```

Comment:
The test should probably be added to the more general `DeletionTests` below as this has little to do with `on_delete` specifically. Also we usually avoid ticket reference except for obscure issues, looks like this test could be renamed `test_delete_sets_pk_none`.